Source/carthage/Extensions.swift

```diff
@@ -108,6 +109,7 @@ internal func waitOnCommand<T>(producer: SignalProducer<T, CommandError>) -> Res
 		|> then(SignalProducer<(), CommandError>.empty)
 		|> wait
 	
+	TaskDescription.waitForAllTaskTermination()
```

Comment:
The segmentation fault happens while disposing after `completed` event from launch task's thread and main thread calling `exit()` concurrently.
Waiting per-task will need waiting some RAC event. That will be same what just current implementation is doing.
I think fixing the segmentation fault needs doing outside of RAC event, and waiting per-task without RAC event may be too complicated unnecessarily.